pkg/util/eventfilter/eventfilter.go

```diff
@@ -35,5 +47,32 @@ func SpecificationChanged(oldObj, newObj *unstructured.Unstructured) bool {
 		unstructured.RemoveNestedField(newBackup.Object, r...)
 	}
 
+	// If Policy LazyReconcile feature enabled, ignore the label/annotation modifications by karmada's own components.
+	if features.FeatureGate.Enabled(features.PolicyLazyReconcile) {
+		// Remove the labels or annotations modified by karmada's own components.
+		for k := range oldBackup.GetLabels() {
```

Comment:
Just a reference, can we extract the deletion of labels and annotations controlled by Karmada into a independent function? This will ensure consistency in the operations of `newBackup` and `oldBackup`.